mffpy/writer.py

```diff
@@ -39,6 +39,7 @@ def __init__(self, filename: str):
         self.file_created = False
 
     def create_directory(self):
+        """Creates the directory for the recording."""
```

Comment:
Since the .mff directory contains a lot of extra info files besides just the signals, we might want to edit the description to say we are creating the directory for all parts of the .mff file.